project/Component/NefCommon/Models/RenderingURL.swift

```diff
@@ -10,6 +10,7 @@ public struct RenderingURL {
 
     public var escapedTitle: String {
         title.lowercased().replacingOccurrences(of: "?", with: "-")
+                          .replacingOccurrences(of: "→", with: "")
```

Comment:
I have had troubles in the past with titles containing `:`, so it'd be nice to add them here. In fact, we should probably consider replacing any non-alphabetical or non-numeric character to ensure we always have a valid file name.

Reply:
Not sure, why I did not do it before, but it makes sense :)